frontend/.eslintrc.js

```diff
@@ -29,7 +29,12 @@ module.exports = {
         destructuredArrayIgnorePattern: "^_",
       },
     ],
-    "@typescript-eslint/consistent-type-imports": "error",
+    "@typescript-eslint/consistent-type-imports": [
+      "error",
+      {
+        fixStyle: "inline-type-imports",
+      },
+    ],
```

Comment:
Oh I didn't realize this was an option, this is nicer imo! Once this is merged let's get a separate PR to apply this style to the rest of the codebase.